Source/DiabloUI/diabloui.cpp

```diff
@@ -734,7 +734,7 @@ void Render(const UiArtText *uiArtText)
 void Render(const UiImage *uiImage)
 {
 	int x = uiImage->m_rect.x;
-	if ((uiImage->m_iFlags & UIS_CENTER) != 0 && uiImage->m_art != nullptr) {
+	if ((uiImage->m_iFlags && UiFlags::UIS_CENTER) && uiImage->m_art != nullptr) {

```

Comment:
u sure u meant `&&`?

Reply:
Yeah this seams highly misleading, I'm not in favor of using this overload.

Reply:
For reference, C# defines a `.HasFlag(flag)` method on enums (seems like an intuitive API to me).
https://docs.microsoft.com/en-us/dotnet/api/system.enum.hasflag?view=net-5.0

Reply:
`HasAnyOf(...)`

Reply:
C++ enums aren't classes (even though scoped enums use the class keyword...) and can't have member functions, which makes it a little awkward to have functions like the C# `HasFlag` equivalent. I've added a commit replacing `operator&&` with `hasAnyOf(UiFlag, UiFlag)` as the closest approximation without a wrapper class.

I'm not sure `std::bitset` helps in this regard, testing for multiple flags requires longer syntax. I'll give it a try this evening and see how it looks.